Evaluation/ExpressionEvaluation.cs

```diff
@@ -21,8 +21,35 @@ public static string EvaluatePostFix(string exp)
             {
                 // If character is an operator, do operation, otherwise put it on the stack
                 char ch = exp[j];
-                if(Operators.Contains(ch))
-                    stack.Push(Evaluate(Convert.ToInt32(stack.Pop()), Convert.ToInt32(stack.Pop()), ch));
+                if (Operators.Contains(ch))
+                    stack.Push(Evaluate(Convert.ToDouble(stack.Pop()), Convert.ToDouble(stack.Pop()), ch));
+                else
```

Comment:
Why the switch to double?

Reply:
if you see, SNO# 5 is 3/2 which returning just 1 for would not be an accurate answer

Reply:
ahhh cheeky okay true